### PR TITLE
[DOC] Adding sections to paper

### PR DESCRIPTION
**Pull request by adam2392**
Thanks for contributing. If this is your first time,
make sure to read the [contributing guide.](https://github.com/mne-tools/mne-python/blob/main/CONTRIBUTING.md)

PR Description
--------------

Closes: #83 

Also fixes: #86 

cc: @TomDonoghue 

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -43,15 +43,17 @@ bibliography: paper.bib
...
+Scalp electroencephalography (EEG) and magnetoencephalography (MEG) analysis is typically very noisy and contains various non-neural signals, such as heart beat artifacts. Independent component analysis (ICA) is a common procedure to remove these artifacts [@Bell1995]. However, removing artifacts requires manual annotation of ICA components, which is subject to human error and very laborious when operating on large datasets. This work makes the popular ICLabel model [@iclabel2019] available in Python by creating a software package compatible with MNE-Python [@Agramfort2013] software toolkit and a modern Pytorch format [@Pytorch2019]. The ICLabel model was previously only available in an outdated version of tensorflow that was no longer supported, and migrating the model now to an updated Pytorch version will ensure the model will not break due to unmaintained versions of software. This enables the automatic labeling of ICA components, improving the preprocessing and analysis pipeline of electrophysiological data.
```
Comment (mscheltienne) [inline paper/paper.md:46]:
```suggestion
Scalp electroencephalography (EEG) and magnetoencephalography (MEG) analysis is typically very noisy and contains various non-neural signals, such as heartbeat artifacts. Independent component analysis (ICA) is a common procedure to remove these artifacts [@Bell1995]. However, removing artifacts requires manual annotation of ICA components, which is subject to human error and very laborious when operating on large datasets. This work makes the popular ICLabel model [@iclabel2019] available in Python by creating a software package compatible with MNE-Python [@Agramfort2013] software toolkit in a modern Pytorch format [@Pytorch2019]. The ICLabel model was previously only available in an outdated version of TensorFlow that was no longer supported, and migrating the model now to an updated Pytorch version will ensure the model will not break due to unmaintained versions of software. This enables the automatic labeling of ICA components, improving the preprocessing and analysis pipeline of electrophysiological data.
```

File: paper/paper.md [hunk 2]
```diff
@@ -43,15 +43,17 @@ bibliography: paper.bib
...
+ICLabel was a proposed statistical model that uses neural networks and a crowdsourced training dataset to automatically label ICA components [@iclabel2019]. However, this package was previously only available in MATLAB, limiting its usage among Python neuroscience researchers. Moreover, the package relied on an outdated version of Tensorflow [@Tensorflow2016], which makes the model difficult to build on. Pytorch is the most common Python deep-learning framework among researchers. Replicating the model in Pytorch enables future automatic ICA annotation research to easily build upon the model.
```
Comment (mscheltienne) [inline paper/paper.md:54]:
```suggestion
ICLabel was a proposed statistical model that uses neural networks and a crowdsourced training dataset to automatically label ICA components [@iclabel2019]. However, this package was previously only available in MATLAB, limiting its usage among Python neuroscience researchers. Moreover, the package relied on an outdated version of Tensorflow [@Tensorflow2016], which makes the model difficult to build on. PyTorch is the most common Python deep-learning framework among researchers. Replicating the model in PyTorch enables future automatic ICA annotation research to easily build upon the model.
```

Comment (mscheltienne) [inline paper/paper.md:54]:
```suggestion
ICLabel was a proposed statistical model that uses neural networks and a crowdsourced training dataset to automatically label ICA components [@iclabel2019]. However, this package was previously only available in MATLAB, limiting its usage among Python neuroscience researchers. Moreover, the package relied on an outdated version of Tensorflow [@Tensorflow2016], which makes the model difficult to build on. PyTorch is the most common Python deep-learning framework among researchers. Replicating the model in PyTorch enables future automatic ICA annotation research to easily build upon the model.
```

File: paper/paper.md (hunk 1 above)
Comment (mscheltienne) [inline paper/paper.md:46]:
```suggestion
Scalp electroencephalography (EEG) and magnetoencephalography (MEG) analysis is typically very noisy and contains various non-neural signals, such as heartbeat artifacts. Independent component analysis (ICA) is a common procedure to remove these artifacts [@Bell1995]. However, removing artifacts requires manual annotation of ICA components, which is subject to human error and very laborious when operating on large datasets. This work makes the popular ICLabel model [@iclabel2019] available in Python by creating a software package compatible with MNE-Python [@Agramfort2013] software toolkit in a modern PyTorch format [@Pytorch2019]. The ICLabel model was previously only available in an outdated version of TensorFlow that was no longer supported, and migrating the model now to an updated PyTorch version will ensure the model will not break due to unmaintained versions of software. This enables the automatic labeling of ICA components, improving the preprocessing and analysis pipeline of electrophysiological data.
```

**Review by mscheltienne — APPROVED** [review]

**Comment by adam2392** [comment]
Rendered docs will be here: https://circleci.com/gh/mne-tools/mne-icalabel/562?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by adam2392** [comment]
https://output.circle-artifacts.com/output/job/c1c5314e-dfc5-4c60-9d70-b49a9b6cdc2c/artifacts/0/dev/api.html

**Comment by adam2392** [comment]
Kay once this is merged, I think the JOSS reviews will be addressed. Lmk if you have any final touches to add @mscheltienne @jacobf18 @anandsaini024

File: doc/index.rst [hunk 3]
```diff
@@ -4,6 +4,15 @@
 mne-icalabel is a Python package for labeling independent components that stem
 from an `Independent Component Analysis (ICA) <https://en.wikipedia.org/wiki/Independent_component_analysis>`_.
 
+Scalp electroencephalography (EEG) and magnetoencephalography (MEG) analysis is typically very noisy
+and contains various non-neural signals, such as heart beat artifacts. Independent
```
Comment (mscheltienne) [inline doc/index.rst:8]:
```suggestion
and contains various non-neural signals, such as heartbeat artifacts. Independent
```

**Review by mscheltienne — APPROVED** [review]

**Pull request merged**